tests/generic_config_updater/test_ipv6.py

```diff
@@ -0,0 +1,196 @@
+    pytest_assert(not neighbor_exists, "Failed to remove ipv6 BGP neighbor under test")
+
+    json_patch = [
+        {
+            "op": "add",
+            "path": "/BGP_NEIGHBOR/{}".format(ipv6_neighbor_address),
+            "value": {
+                "admin_status": "up",
+                "asn": "{}".format(ipv6_neighbor_config['asn']),
+                "holdtime": "{}".format(ipv6_neighbor_config['holdtime']),
+                "keepalive": "{}".format(ipv6_neighbor_config['keepalive']),
+                "local_addr": "{}".format(ipv6_neighbor_config['local_addr']),
+                "name": "{}".format(ipv6_neighbor_config['name']),
+                "nhopself": "{}".format(ipv6_neighbor_config['nhopself']),
+                "rrclient": "{}".format(ipv6_neighbor_config['rrclient'])
+            }
+        }
+    ]
+    
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try: 
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+        neighbor_exists = check_neighbor_existence(duthost, ipv6_neighbor_address)
+        pytest_assert(neighbor_exists, "GCU failed to add back deleted ipv6 BGP neighbor")
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+
+def test_delete_ipv6_neighbor(duthost, ensure_dut_readiness):
+    ipv6_neighbor_address, ipv6_neighbor_config = get_ipv6_neighbor(duthost)
+
+    json_patch = [
+        {
+            "op": "remove",
+            "path": "/BGP_NEIGHBOR/{}".format(ipv6_neighbor_address)
+        }
+    ]
+
+    tmpfile = generate_tmpfile(duthost)
+    logger.info("tmpfile {}".format(tmpfile))
+
+    try:
+        output = apply_patch(duthost, json_data=json_patch, dest_file=tmpfile)
+        expect_op_success(duthost, output)
+        neighbor_exists = check_neighbor_existence(duthost, ipv6_neighbor_address)
+        pytest_assert(not neighbor_exists, "Failed to remove ipv6 BGP neighbor under test")
+    finally:
+        delete_tmpfile(duthost, tmpfile)
+
+
+def test_add_duplicate_ipv6_neighbor(duthost, ensure_dut_readiness):
+    ipv6_neighbor_address, ipv6_neighbor_config = get_ipv6_neighbor(duthost)
+
+    json_patch = [
+        {
+            "op": "add",
+            "path": "/BGP_NEIGHBOR/{}".format(ipv6_neighbor_address),
+            "value": {
+                "admin_status": "{}".format(ipv6_neighbor_config['admin_status']),
+                "asn": "{}".format(ipv6_neighbor_config['asn']),
+                "holdtime": "{}".format(ipv6_neighbor_config['holdtime']),
+                "keepalive": "{}".format(ipv6_neighbor_config['keepalive']),
+                "local_addr": "{}".format(ipv6_neighbor_config['local_addr']),
+                "name": "{}".format(ipv6_neighbor_config['name']),
+                "nhopself": "{}".format(ipv6_neighbor_config['nhopself']),
+                "rrclient": "{}".format(ipv6_neighbor_config['rrclient'])
```

Comment:
Could we just use `ipv6_neighbor_config` instead of manually construct each pair?